### PR TITLE
Ehcache3 custom serializers blog

### PR DESCRIPTION
**Pull request by albinsuresh**

### CODE REVIEW

File: _posts/blog/2016-05-10-ehcache3-serializers.adoc [hunk 1]
```diff
@@ -0,0 +1,673 @@
+---
+layout: post
+title: "Ehcache3 Custom Serializers"
+headline: "Plug in your custom serialization technique seamlessly with Ehcache3"
```
Comment (albinsuresh) [inline _posts/blog/2016-05-10-ehcache3-serializers.adoc]: Am not quite happy with this headline. Suggestions welcome.

Comment (ljacomet) [inline _posts/blog/2016-05-10-ehcache3-serializers.adoc]: Proposition: "Understand and leverage the serialization subsystem in Ehcache 3"

File: _posts/blog/2016-05-10-ehcache3-serializers.adoc [hunk 2]
```diff
@@ -0,0 +1,673 @@
+---
+layout: post
+title: "Ehcache3 Custom Serializers"
```
Comment (ljacomet) [inline _posts/blog/2016-05-10-ehcache3-serializers.adoc]: Ehcache 3
(add the space)

File: _posts/blog/2016-05-10-ehcache3-serializers.adoc [hunk 3]
```diff
@@ -0,0 +1,673 @@
+---
+layout: post
+title: "Ehcache 3 Custom Serializers"
+headline: "Understand and leverage the serialization subsystem in Ehcache 3"
+categories: blog
+hidden: false
+author: albin_suresh
+tags: [ehcache, serializers]
+date: 2016-05-10T17:35:00+05:30
```
Comment (ljacomet) [inline _posts/blog/2016-05-10-ehcache3-serializers.adoc]: May 12 and adapt the file name ... then I merge it today :wink:

**Pull request merged**